### ISSUE TITLE
Instructions are incomplete and hard to follow

### ISSUE THREAD

**Issue by matthewjlevy** [AzureAD/Deployment-Plans#15]
I am trying to follow the instructions in the readme.md file for the ADFSAADMigrationUtils and I have run the 3 cmdlets to generate 5 CSV files. The instructions then say "From this same workstation, open **_this Excel spreadsheet_** and navigate to the Dashboard tab and hit the Refresh Data button on the right."

What is the name of "this Excel spreadsheet"?

**Comment by matthewjlevy** [comment]
Hi there, any update on this issue?

**Comment by weedee77** [comment]
Matthew I'm wondering this same thing. Maybe others have found this piece of information elsewhere.

**Comment by robin-martin** [comment]
This file spreadsheet
ADFS to AAD App Migration Report Template.xlsm
